pkgs/development/libraries/vtk/generic.nix

```
@@ -25,6 +27,7 @@
   enablePython ? false,
   python ? throw "vtk: Python support requested, but no python interpreter was given.",
   enableEgl ? false,
+  enableVtkDicom ? true,
```

If a solution can not be found/tested, consider:
```suggestion
	# fatal error: 'sqlite3.h' file not found
  enableVtkDicom ? !stdenv.hostPlatform.isDarwin,
```